recipe/meta.yaml

```diff
@@ -11,29 +12,53 @@ source:
     - have_openssl.patch
 
 build:
-  number: 2
-
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - postgresql                             # [build_platform != target_platform]
-    - {{ compiler('c') }}
-  host:
-    - python
-    - pip
-    - openssl  # [not linux]
-    - postgresql
-  run:
-    - python
-    # libpq is a repackaging of only the library to connect to postgres
-    - libpq
-    - openssl  # [not linux]
-
-test:
-  imports:
-    - psycopg2
-    - psycopg2._psycopg
+  number: 3
+
+outputs:
+  - name: {{ name }}
+    script: install_psycopg2.sh  # [unix]
+    script: install_psycopg2.bat  # [win]
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - postgresql                             # [build_platform != target_platform]
+        - {{ compiler('c') }}
+      host:
+        - python
+        - pip
+        - openssl  # [not linux]
+        - postgresql
+      run:
+        - python
+        # libpq is a repackaging of only the library to connect to postgres
+        - libpq
+        - openssl  # [not linux]
+
+    test:
+      imports:
+        - psycopg2
+        - psycopg2._psycopg
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  # alternative name for the package
+  - name: psycopg2-binary
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
+    test:
+      imports:
+        - psycopg2
+      commands:
+        - pip check
```

Comment:
can you add some downstream to `requires` that requires `psycopg2-binary`? I don't _really_ understand how naming something on the conda side would actually fake out the metadata checks... potentially making `psycopg2` a `host` dependency and copying/patching `some-email-header-formatted-file.txt` might do the thing...

Reply:
Oh, I see.  You are right.  This won't change anything for `pip check`.  It would just make `psycopg2-binary` available as a `conda package`, which is not what is needed.  Shoot!